---
title: Research
permalink: /research/
---
<img src="{{site.baseurl}}/images/research/research_structure.png" data-action="zoom">
Our lab works on bioinformatics, mass spectrometry, and proteomics technologies as part of the Department of Biomedical Informatics and the Pelotonia Institute for Immuno-Oncology at the Ohio State University. We are interested in developing new methods that improve all aspects of the proteomics workflow, from data acquisition, to computational quantitation and statistical analysis. We are dedicated to open methods and open source software, as well as the long-term support for those computational resources for the proteomics and mass spectrometry communities.

### Human health and disease

|-----|-----|
|<img width="500" src="{{site.baseurl}}/images/research/hla_variants.png" data-action="zoom">|<label style="font-weight: 300 !important; width: 400px;">We work on using proteomics to characterize the effects of genetic variation on proteins and protein function, with specific interest in variants that help drive cancer.<br><br>For example, in this figure we can identify single amino acid polymorphisms in HLA:<br><b>(a)</b> Fragment ions detected in HLA-A (compared to the same peptides in HLA-B) indicate sequence variants in HeLa. <br><b>(b)</b> These variants are in the peptide-binding region and likely affect the binding and presentation of antigens to the immune system. <br>Figure modified from [Searle et al Nat Comm 2020](https://www.nature.com/articles/s41467-020-15346-1)<br><br>Following this approach for mining the proteome, we are developing new software techniques to detect, quantify, and track the effects of genetic varation at the protein-level.</label>|

### Mass spectrometry

|-----|-----|
|<img width="500" src="{{site.baseurl}}/images/research/dia_basics.png" data-action="zoom">|<label style="font-weight: 300 !important; width: 400px;">Our lab makes tools for Data Independent Acquisition Mass Spectrometry (DIA-MS). Using DIA-MS, multiple peptides are separated across retention time with liquid chromatography. Peptide precursors in a similar mass range are selected in an isolation window (green highlight) to be co-fragmented. By lining up co-eluting fragment ions across retention time, researchers assign specific fragment ions to individual analytes. <br><br>In our lab, we manipulate the mass spectrometer to better collect and analyze this type of data in ways that increase consistency and proteome depth. We have made several resources for implementing DIA-MS in your lab, including our [MCP tutorial paper](https://doi.org/10.1074/mcp.P119.001913) by Lindsay Pino and various Youtube videos: [(1)](https://www.youtube.com/watch?v=RidYXjvAk0s), [(2)](https://www.youtube.com/watch?v=EjHyMxIOMKE), and [(3)](https://www.youtube.com/watch?v=coRDFs8Z8Ks).</label>|

### Bioinformatics

|-----|-----|
|<img width="500" src="{{site.baseurl}}/images/research/neuralnetwork.png" data-action="zoom">|<label style="font-weight: 300 !important; width: 400px;">We use bioinformatics as a key component of all of our work, from developing low-level software for extracting signals from mass spectrometry experiments to building scripts to statistically interpret our biological data. Our focus is on building industry-strength, robust software that broadly enable the research community. You can read more about our current and maintenance supported software projects [here](../software/). Finally, all software projects we build are released under liberal, open source licenses.</label>|
